core/descriptors/src/org/jetbrains/kotlin/descriptors/impl/SimpleFunctionDescriptorImpl.java

```diff
@@ -53,6 +53,7 @@ public static SimpleFunctionDescriptorImpl create(
     @NotNull
     @Override
     public SimpleFunctionDescriptorImpl initialize(
+            @NotNull List<ReceiverParameterDescriptor> additionalReceiverParameters,
```

Comment:
I'd move this parameter after common receivers, i.e. made it the third one

Reply:
Fixed